### PR TITLE
IA-1998: provide planning filter forms list frontend

### PR DESCRIPTION
**Pull request by codeKameleon**
We couldn't filter forms list result by related planning

Related JIRA tickets : IA-1497 / IA-1998

## Self proofreading checklist

- [x] Did I use eslint and black formatters
- [x] Is my code clear enough and well documented
- [x] Are my typescript files well typed
- [ ] New translations have been added or updated if new strings have been introduced in the frontend
- [x] My migrations file are included
- [ ] Are there enough tests
- [ ] Documentation has been included (for new feature)

## Changes

- add planning to params
- split the results of planning filter to sort on multiple plannings

## How to test

- Go to forms
- Play with the Planning filter, see if the results are correct

## Print screen / video

https://github.com/BLSQ/iaso/assets/25134301/e1d76f45-3684-4f04-8d77-e8fb6c6bc0e3

### CODE REVIEW

File: hat/assets/js/apps/Iaso/domains/forms/components/Filters.tsx [hunk 1]
```diff
@@ -44,6 +46,7 @@ const Filters: FunctionComponent<Props> = ({ params }) => {
     const [showDeleted, setShowDeleted] = useState<boolean>(
         filters.showDeleted === 'true',
     );
+    const { data: planningsDropdown } = useGetPlanningsOptions();
```
Comment (beygorghor) [inline hat/assets/js/apps/Iaso/domains/forms/components/Filters.tsx]: `planningsDropdownOptions` ?

File: iaso/api/forms.py [hunk 2]
```diff
@@ -226,7 +226,7 @@ def get_queryset(self):
 
         planning_id = self.request.query_params.get("planning", None)
```
Comment (beygorghor) [inline iaso/api/forms.py]: i would call it `planning_ids` then

**Review by beygorghor — CHANGES_REQUESTED** [review]
i have an issue with the search button:
Screenshot 2023-06-01 at 11 54 25

**Comment by beygorghor** [comment]
Also you have a conflict and js test are failing

**Comment by codeKameleon** [comment]
@cypress

File: hat/assets/js/cypress/integration/02 - forms/list.spec.js [hunk 3]
```diff
@@ -195,7 +197,7 @@ describe('Forms', () => {
         });
     });
 
-    describe('api', () => {
+    describe.only('api', () => {
```
Comment (beygorghor) [inline hat/assets/js/cypress/integration/02 - forms/list.spec.js]: I'll fix that

**Review by beygorghor — APPROVED** [review]
just a .only to remove, i'll do it

**Pull request merged**